modopt/tests/test_signal.py

```diff
@@ -1,322 +1,233 @@
-# -*- coding: utf-8 -*-
-
 """UNIT TESTS FOR SIGNAL.
 
 This module contains unit tests for the modopt.signal module.
 
-:Author: Samuel Farrens <samuel.farrens@cea.fr>
-
+:Authors:
+    Samuel Farrens <samuel.farrens@cea.fr>
+    Pierre-Antoine Comby <pierre-antoine.comby@cea.fr>
 """
 
-from unittest import TestCase
-
 import numpy as np
 import numpy.testing as npt
+import pytest
+from test_helpers import failparam
 
 from modopt.signal import filter, noise, positivity, svd, validation, wavelet
 
 
-class FilterTestCase(TestCase):
-    """Test case for filter module."""
```

Comment:
Can you explain why the this test case (class) was removed? It doesn't seem to be consistent with the rest of the test cases in the module.

Reply:
I guess by laziness, as the number of test was small I did not bother putting it in a dedicated class (it does not change anything for pytest)